pulsar-client-kafka-compat/pulsar-client-kafka_0_8/src/test/java/org/apache/pulsar/client/kafka/test/KafkaProducerSimpleConsumerTest.java

```diff
@@ -141,6 +141,7 @@ public void testPulsarKafkaProducerWithSerializer(int partitions) throws Excepti
             producer.send(message);
         }
         producer.close();
+        Thread.sleep(500);
```

Comment:
It's better to avoid using sleep in the test, could you please use Awaitility instead?

Reply:
I agree that it would be nice, but I suggest we merge this to unblock the release + add an issue to fix it later (it will take time to figure out what to wait for specifically, especially given that this test failed only on CI and passed on local runs) + there are other tests that rely on sleep: https://github.com/apache/pulsar-adapters/search?q=Thread.Sleep

Reply:
I am fine for keeping Sleep for now.
it is not clear what we have to wait for at the moment